deploy.sh

```
@@ -230,19 +256,44 @@ print_message "$RED" "2) Destroy (BE CAREFUL)"
 
 read -p "Select an action (1-2): " action_choice
 
+# Ask about plugin resources (only for AWS currently)
+include_plugins=false
+if [ "$provider" = "aws" ]; then
+    print_message "$YELLOW" "\nDo you want to include plugin resources?"
+    print_message "$NC" "This will deploy/destroy DocumentDB instances for Fee and CRM plugins"
+    print_message "$GREEN" "1) Yes - Include plugin resources"
+    print_message "$NC" "2) No - Core infrastructure only"
+
+    read -p "Select option (1-2): " plugin_choice
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)